docs/javascripts/imagelightbox.js

```diff
@@ -160,8 +160,56 @@
                     $arrows.css('display', 'block');
                 }
             },
+            _pushHistory = function () {
+                var query = document.location.search;
+                var newParam = 'imageLightboxIndex=' + targetIndex;
+                var newQuery = '?' + newParam;
+
+                if (query) {
+                    var keyRegex = new RegExp('([?&])imageLightboxIndex=[^&]*');
+                    if (query.match(keyRegex) !== null) {
+                        newQuery = query.replace(keyRegex, '$1' + newParam);
+                    } else {
+                        newQuery = query + '&' + newParam;
+                    }
+                }
+                window.history.pushState({imageLightboxIndex: targetIndex}, '', newQuery);
+            },
+            _pushHistoryQuit = function () {
```

Comment:
so the function pushes and quits?

Reply:
No, it pushes the quiting of the lightbox to the history. If you have a better ( = more descriptive ) name, I'd be glad to use it

Reply:
maybe pushQuitToHistory?